### PR TITLE
Feature/ongoing exploration

### PR DESCRIPTION
**Pull request by smritiDivami**
# TITLE

## Description
Summary of changes

## PR Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [ ] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: [Jira issue #](fill-in.org)

### CODE REVIEW

File: app/cdap/components/ConnectorTypes/index.tsx [hunk 1]
```diff
@@ -34,23 +34,28 @@ const ConnectorTypesComponent = () => {
...
-                image={eachConnectorType.image}
-              />
-            </Link>
-          ))}
+      <Paper className={classes.wrangleHomeContainer}>
+        <Paper variant="outlined" elevation={9} className={classes.dashBoard}>
+          <WelcomeCard />
+          <Typography className={classes.subTitle}>Start data exploration</Typography>
+          <Box className={classes.underLine}>{UnderLine}</Box>
+          <Paper elevation={0} className={classes.flexContainer}>
```
Comment (adityadivami) [inline app/cdap/components/ConnectorTypes/index.tsx:42]: Use MUI Grid Component for alignment of connector types component and on going exploarations component

File: app/cdap/components/OngoingDataExploration/LinearProgressBar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { StyledLinearProgress } from './styles';
+
+const LinearProgressComponent = ({ progressValue }) => {
+  return <StyledLinearProgress variant="determinate" value={progressValue} />;
```
Comment (adityadivami) [inline app/cdap/components/OngoingDataExploration/LinearProgressBar/index.tsx]: Send variant and value from caller component
Make this component generic {...props}

give interface appropriately

File: app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/CardContext.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      return (
+        <Typography component="div" variant="body1" className={styles.datasetName}>
+          {item.label}
+        </Typography>
+      );
+    case 'subLabel':
+      return (
+        <Typography component="div" variant="body2" className={styles.datasetDetails}>
+          {item.label}
+          {item.subText}
```
Comment (adityadivami) [inline app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/CardContext.tsx]: use \&nbsp;\ for space

File: app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import { Card, CardContent } from '@material-ui/core';
+import { CardContentComponent } from './CardContext';
+import { explorationCardStyles } from './styles';
+
+const OngoingExplorationCard = ({ itemData }) => {
+  const cardData = itemData;
+  const classes = explorationCardStyles();
+
+  return (
+    <>
```
Comment (adityadivami) [inline app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/index.tsx]: fragment is not necessary

File: app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/index.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+      <Card className={classes.explorationCardStyles}>
+        <CardContent className={classes.cardContent}>
+          {cardData.map((eachItem) => {
```
Comment (adityadivami) [inline app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/index.tsx]: add proper checks for cardData
no need to use cardData, we can directly use itemData

File: app/cdap/components/OngoingDataExploration/index.tsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+const OngoingDataExploration = () => {
+  const classes = useStyles();
+  return (
+    <>
+      <Box className={classes.rightContainerStyles}>
+        <Typography component="div" variant="body2" className={classes.dataExplorationHeader}>
+          Continue ongoing data explorations, pick up where you left off
+        </Typography>
+        <Box className={classes.underLine}>{UnderLine}</Box>
+        <Box className={classes.dataExplorationCardContainer}>
```
Comment (adityadivami) [inline app/cdap/components/OngoingDataExploration/index.tsx]: Use Grid container and Grid for flex

File: app/cdap/components/OngoingDataExploration/mock.ts [hunk 7]
```diff
@@ -0,0 +1,60 @@
+export const mockData = [
```
Comment (adityadivami) [inline app/cdap/components/OngoingDataExploration/mock.ts]: addd expected interface

also move mock to a folder and keep index and types inside the folder

**Review by adityadivami — COMMENTED** [review]
Please resolve the comments and 
1. add interfaces appropriately everywhere

File: app/cdap/components/ConnectorTypes/index.tsx (hunk 1 above)
Comment (smritiDivami) [reply app/cdap/components/ConnectorTypes/index.tsx:42]: "Added Grid to Connector Types Component."

File: app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/index.tsx (hunk 4 above)
Comment (smritiDivami) [reply app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/index.tsx]: removed <>

File: app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/CardContext.tsx (hunk 3 above)
Comment (smritiDivami) [reply app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/CardContext.tsx]: added "nbsp;"

File: app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/index.tsx (hunk 5 above)
Comment (smritiDivami) [reply app/cdap/components/OngoingDataExploration/OngoingDataExplorationCard/index.tsx]: removed cardData.

**Pull request closed without merging**